pages/content/closed-captioning.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: primary
+title: Video captioning
```

Comment:
can we change this to: "Video and multimedia"

Reply:
Sure. Should probably reflect that in the URLs as well, right?

Reply:
Done in 5284cce and 9da2627.